ruby_connectors/spec/temporal/activities/read_api_data_activity_spec.rb

```
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe RubyConnectors::Temporal::Activities::ReadApiDataActivity do
```

rubocop (1.68.0)

[convention] Spec path should end with `ruby_connectors/temporal/activities/read_api_data_activity*_spec.rb`.

(RSpec/SpecFilePathFormat)